### ISSUE TITLE
Hard coding API keys is a bad practice, you should keep them as ENV variables

### ISSUE THREAD

**Issue by Victornovikov** [NicholasTagher/SportsCalendar#1]
https://github.com/NicholasTagher/SportsCalendar/blob/b7d4f8509baa3c3d615ae16f14e63fb6a666ec4f/main.py#L14

**Comment by Victornovikov** [comment]
since it is a public repo you should probably rotate your keys now..

**Comment by Victornovikov** [comment]
same goes for PII https://github.com/NicholasTagher/SportsCalendar/blob/b7d4f8509baa3c3d615ae16f14e63fb6a666ec4f/main.py#L134

**Comment by NicholasTagher** [comment]
Yes, they sent me a message right away telling me the keys were changed. I
have to update it

**Comment by NicholasTagher** [comment]
What is an ENV variable? I thought it was such, because I store it then
feed the variable into the Client field?

**Comment by Victornovikov** [comment]
https://www.twilio.com/blog/environment-variables-python

**Comment by NicholasTagher** [comment]
Hi, so I uploaded the new files by using functions. I also used the ENV
variables
